src/libpriv/rpmostree-bwrap.cxx

```diff
@@ -429,6 +429,35 @@ rpmostree_bwrap_take_fd (RpmOstreeBwrap *bwrap,
   g_subprocess_launcher_take_fd (bwrap->launcher, source_fd, target_fd);
 }
 
+void 
+rpmostree_bwrap_take_stdin_fd (RpmOstreeBwrap *bwrap,
+                              int             source_fd)
+{
+  g_subprocess_launcher_take_stdin_fd (bwrap->launcher, source_fd);
+}
```

Comment:
These thin wrappers for the `g_subprocess_launcher()` APIs are annoying but they'll be easier to maintain when `rpmostree-bwrap.cxx` migrates to Rust.